doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -369,167 +373,173 @@ throttling
 .. confval:: bluestore_throttle_cost_per_io_hdd
 .. confval:: bluestore_throttle_cost_per_io_ssd
 
-spdk usage
-==================
+SPDK Usage
+==========
 
-if you want to use the spdk driver for nvme devices, you must prepare your system.
-refer to `spdk document`__ for more details.
+To use the SPDK driver for NVMe devices, you must first prepare your system.
+See `SPDK document`__.
 
 .. __: http://www.spdk.io/doc/getting_started.html#getting_started_examples
 
-spdk offers a script to configure the device automatically. users can run the
-script as root:
+SPDK offers a script that will configure the device automatically. Run this
+script with root permissions:
 
 .. prompt:: bash $
 
    sudo src/spdk/scripts/setup.sh
 
-you will need to specify the subject nvme device's device selector with
-the "spdk:" prefix for ``bluestore_block_path``.
+You will need to specify the subject NVMe device's device selector with the
+"spdk:" prefix for ``bluestore_block_path``.
 
-for example, you can find the device selector of an intel pcie ssd with:
+In the following example, you first find the device selector of an Intel PCIe
+SSD by running the following command:
```

Comment:
s/PCIe/NVMe/

We call it NVMe elsewhere, and NVMe more or less implies PCI-e

Reply:
Accepted.